Test/verification/filter.dfy.expect

```diff
@@ -0,0 +1,6 @@
+
+Dafny program verifier finished with 1 verified, 0 errors
+filter.dfy(10,16): Error: A postcondition might not hold on this return path.
+filter.dfy(10,10): Related location: This is the postcondition that might not hold.
+
+Dafny program verifier finished with 1 verified, 1 error
```

Comment:
Isn't this result backwards? The first call filters to `Success` so it should succeed AFAICT

Reply:
The double quotes made our custom lit parser fail and ignore the argument.

Reply:
Never mind, I was confused by the spacing of the output. The first two lines are from the successful run from filtering to the `Success`, and the following errors are from the second run that doesn't filter.

I do see that the custom lit parser is dropping the single quotes in the argument, but that doesn't seem to affect correctness here. Might be worth fixing for maximum consistency in the future though.